packages/wonder-blocks-button/components/button.md

```diff
@@ -120,6 +120,8 @@ const styles = StyleSheet.create({
 </View>
 ```
 
+**Note:** If you are adding `disabled` you don't need to add `aria-disabled`. Doing so will result in a console.warn().
```

Comment:
It might be worth adding a bit more context here to help people understand *why* this is undesirable.

Reply:
It's not just that you don't need it, it's that it can confuse the screen readers, I believe?